manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -837,6 +1181,33 @@ spec:
       gatekeeper.sh/system: "yes"
 ---
 apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
```

Comment:
Since mutation feature is still alpha, let's not add it to this gatekeeper.yaml just yet.

We have created this https://github.com/open-policy-agent/gatekeeper/blob/master/deploy/experimental/gatekeeper-mutation.yaml for mutation deployment yaml.